portfolio.html

```diff
@@ -0,0 +1,121 @@
+<!DOCTYPE HTML>
+
+<html>
+
+<head>
+	<link rel="stylesheet" type="text/css" href="https://necolas.github.io/normalize.css/8.0.1/normalize.css">
+	<link rel="stylesheet" type="text/css" href="portfolio.css">
+	<title>Amyas Gilbert's Portfolio</title>
+	<!-- <meta charset="UTF-8"> -->
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+</head>
+
+<body>
+
+	<nav>
+		<div class="nav-buttons">
+			<a href="#about">About</a>
+			<a href="#portfolio">Portfolio</a>
+			<a href="#contact">Contact</a>
+		</div>
+	</nav>
+	
+	<header>
+		<img class="tardi-image" src="tardi-dude.png">		
+		<div class="header-text">
+			<h1>Greetings!</h1>
+			<p class="intro-text">My name is Amyas and I'm a Full Stack Developer</p>
+			<p class="peruse">Please peruse my portfolio ✨</p>
```

Comment:
peruse is not really a semantic name for this class, although it is charming 😅 luckily it is actually not used for anything, so you can remove it